packages/sign-client/test/sdk/validation.spec.ts

```diff
@@ -29,7 +29,6 @@ describe("Sign Client Validation", () => {
     let clients: Clients;
     beforeAll(async () => {
       clients = await initTwoClients();
-      await testConnectMethod(clients);
```

Comment:
Nice optimisation ✨ Seems obvious in hindsight that the validation tests don't need any existing connection but never noticed this.